Assets/Scripts/GlyphRecognition.cs

```diff
@@ -13,6 +13,12 @@ public class GlyphRecognition : MonoBehaviour {
 
 	public StrokeGraphic targetGlyphGraphic, castedGlyphGraphic, currentGlyphGraphic, currentStrokeGraphic, storedGlyphGraphic;
 
+	float costThreshold;
+	public float cstThreshold {get{ return costThreshold; } set{
```

Comment:
As far as I know the style standard for this is to capitalize the first letter ie. CostThreshold